OpenSim/Common/Component.h

```diff
@@ -1256,9 +1256,15 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Component, Object);
 
     /**
      * %Set all values of the state variables allocated by this Component.
-     * Includes state variables allocated by its subcomponents.
+     * Includes state variables allocated by its subcomponents. Note, this
+     * method simply sets the values on the input State. If other conditions
+     * must be met (such as fiber and tendon equilibrium for muscles, or to
+     * satisfy kinematic constraints for Coordinates) you must invoke the
+     * appropriate methods on Model (e.g. assemble() to satisfy constraints or
+     * equilibrateMuscles()) to satisfy these conditions starting from the
+     * State values provided by setStateVariableValues.
      *
-     * @param state   the State for which to get the value
+     * @param state   the State for which to set State values upon
```

Comment:
Rearranged, this description is "The State to set State values upon for [sic]". Perhaps "The State whose values are set"?